src/main/java/com/solvd/noteservice/service/impl/NoteServiceImpl.java

```diff
@@ -17,55 +20,86 @@
 
     private final NoteRepository noteRepository;
+    private final ElasticClient elasticClient;
     private final RestTemplate restTemplate;
+    private final KfProducer kfProducer;
 
     @Override
-    public boolean isExistById(Long id) {
-        return noteRepository.existsById(id);
+    public final boolean isExistById(final Long id) {
+        return elasticClient.isExistById(id);
     }
 
     @Override
     @Transactional
-    public Note create(Note note) {
-        Boolean isExistByUserId = restTemplate.getForObject("http://user/api/v1/users/{id}",
+    public final Note create(final Note note) {
+        Boolean isExistByUserId = restTemplate
+                .getForObject("http://user/api/v1/users/{id}",
                 Boolean.class,
                 note.getUserId());
         if (Boolean.FALSE.equals(isExistByUserId)) {
-            throw new ResourceDoesNotExistException("There are no user with id " + note.getUserId());
+            throw new ResourceDoesNotExistException(
+                    "There are no user with id " + note.getUserId()
+            );
         }
-        noteRepository.save(note);
-        return note;
+
+        NoteEvent noteEvent = new NoteEvent();
+        noteEvent.setType(NoteEvent.Method.POST);
+        noteEvent.setId(note.getId());
+        noteEvent.setDescription(note.getDescription());
+        noteEvent.setTheme(note.getTheme());
+        noteEvent.setTag(note.getTag());
+        noteEvent.setUserId(note.getUserId());
+        kfProducer.sendMessage(noteEvent);
+        return noteRepository.save(note);
     }
 
     @Override
-    public List<Note> findAllByUserId(Long userId) {
-        return noteRepository.findAllByUserId(userId);
+    public final List<Note> findAllByUserId(final Long userId) {
+        return elasticClient.findAllByUserId(userId);
     }
 
     @Override
-    public List<Note> findAll() {
-        return noteRepository.findAll();
+    public final List<Note> findAll() {
+        return elasticClient.findAll();
     }
 
     @Override
-    public Note findById(Long id) {
-        Note note = noteRepository.findById(id)
-                .orElseThrow(() -> new ResourceDoesNotExistException("There are no note with id" + id));
+    public final Note findById(final Long id) {
+        Note note = elasticClient.findById(id)
+                .orElseThrow(() -> new ResourceDoesNotExistException(
+                        "There are no note with id" + id)
+                );
         return note;
     }
 
     @Override
-    public Note update(Note note) {
+    public final Note update(final Note note) {
         Note noteFromDb = noteRepository.findById(note.getId()).get();
         if (!noteFromDb.getTheme().equals(note.getTheme())) {
-            throw new IllegalOperationException("You cant change the theme of note");
+            throw new IllegalOperationException(
+                    "You cant change the theme of note"
+            );
         }
+
```

Comment:
@vpetrovaa empty lines it is not a good idea

Reply:
@h1alexbel checkstyle checking marked it as error, ignore this case?